Simplified/NYPLBookContentType.m

```diff
@@ -5,7 +5,8 @@ NYPLBookContentType NYPLBookContentTypeFromMIMEType(NSString *const string)
 {
   if ([string isEqualToString:ContentTypeFindaway] ||
       [string isEqualToString:ContentTypeOpenAccessAudiobook] ||
-      [string isEqualToString:ContentTypeFeedbooksAudiobook]) {
+      [string isEqualToString:ContentTypeFeedbooksAudiobook] ||
+      [string isEqualToString:ContentTypeOverdriveAudiobook]) {
```

Comment:
If we have this many types, we might want to consider checking if the string matches an element in an array, where the array contains audiobook types.